pykrige/variogram_models.py

```diff
@@ -3,50 +3,66 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
-__doc__ = """Code by Benjamin S. Murphy
-bscott.murphy@gmail.com
-
-Dependencies:
-    numpy
-
-Methods:
-
-    linear_variogram_model(m, d):
-        m (array-like): [slope, nugget]
-        d (array-like): Points at which to calculate variogram model.
-
-    power_variogram_model(m, d):
-        m (array-like): [scale, exponent, nugget]
-        d (array-like): Points at which to calculate variogram model.
-
-    gaussian_variogram_model(m, d):
-        m (array-like): [psill, range, nugget]
-        d (array-like): Points at which to calculate variogram model.
+__doc__ = """
+PyKrige
+=======
 
-    exponential_variogram_model(m, d):
-        m (array-like): [psill, range, nugget]
-        d (array-like): Points at which to calculate variogram model.
-
-    spherical_variogram_model(m, d):
-        m (array-like): [psill, range, nugget]
-        d (array-like): Points at which to calculate variogram model.
-
-    hole_effect_variogram_model(m, d):
-        m (array-like): [psill, range, nugget]
-        d (array-like): Points at which to calculate variogram model.
-
-*** NOTE these functions use the partial sill (psill = sill - nugget) rather than the full sill...
-    the PyKrige user interface by default takes the full sill (although this can be changed with a flag),
-    but it's safer to perform automatic variogram estimation using the partial sill
-
-*** ALSO NOTE that Kitanidis says the hole-effect variogram model is only correct for the 1D case...
-    it's implemented here for completeness and should be used cautiously...
-
-References:
-    P.K. Kitanidis, Introduction to Geostatistcs: Applications in Hydrogeology,
-    (Cambridge University Press, 1997) 272 p.
+Code by Benjamin S. Murphy and the PyKrige Developers
+bscott.murphy@gmail.com
 
-Copyright (c) 2015-2017 Benjamin S. Murphy
+Summary
+-------
+Function definitions for variogram models.
+
+Routines
+--------
+linear_variogram_model(m, d)

```

Comment:
I forgot, one never need to call these variogram functions directly, right? 
In this case might be better to move this description directly to the docs?
For instance under `doc/variogram_functions.rst` and add `variogram_functions` to `toctree` in `index.rst`, that way it would be easier to find it.. This could also be a place to discuss more general aspects of variogram functions...

Reply:
Yeah, these never get directly called by the user. Moving the descriptions directly to the docs is a good idea.